code/datums/wires/explosive.dm

```diff
@@ -66,6 +68,8 @@
 	holder_type = /obj/item/grenade/c4
 
 /datum/wires/explosive/c4/interactable(mob/user) // No need to unscrew wire panels on plastic explosives
+	if(!..())
+		return TRUE
 	return TRUE
```

Comment:
????

Reply:
Haha, I did find and replace, so this happened. The parent proc returns true anyway, so this definition is useless

Reply:
I don't even know man... What exactly was I thinking?